Resources/translations/sub_items.en.xliff

```diff
@@ -86,6 +81,71 @@
+      </trans-unit>
+      <trans-unit id="d2ab57f08d00c6f0d714b26aa81a44c8a1867903" resname="bulk_hide.popup.message">
+        <source>Are you sure you want to hide the selected location(s)?</source>
+        <target state="new">Are you sure you want to hide the selected location(s)?</target>
+        <note>key: bulk_hide.popup.message</note>
+      </trans-unit>
+      <trans-unit id="3a8fe14b551e382f3f1f20f7aed2e5de9804f619" resname="bulk_hide.error.modal.table_title">
+        <source>Locations cannot be hidden (%itemsCount%)</source>
+        <target state="new">Locations cannot be hidden (%itemsCount%)</target>
+        <note>key: bulk_hide.error.modal.table_title</note>
+      </trans-unit>
+      <trans-unit id="5e5f346e9dca66d113585207c4e7a98e87421d34" resname="bulk_hide.error.message">
+        <source>%failedCount% of the %totalCount% selected location(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</source>
+        <target state="new">%failedCount% of the %totalCount% selected location(s) could not be hidden because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</target>
+        <note>key: bulk_hide.error.message</note>
+      </trans-unit>
+      <trans-unit id="9157594a9c2f44581165ee5e73c89bc5985a7fc9" resname="bulk_hide.success.message">
+        <source>The selected location(s) have been hidden.</source>
+        <target state="new">The selected location(s) have been hidden.</target>
+        <note>key: bulk_hide.success.message</note>
+      </trans-unit>
+      <trans-unit id="16c240de3cfc07b8e56d008357706abefad3e3ae" resname="bulk_unhide.popup.confirm">
+        <source>Reveal</source>
+        <target state="new">Reveal</target>
+        <note>key: bulk_unhide.popup.confirm</note>
+      </trans-unit>
+      <trans-unit id="99b9c21ff0ed8b33521c0b8c07410f1f10e5f879" resname="bulk_unhide.popup.message">
+        <source>Are you sure you want to reveal the selected location(s)?</source>
+        <target state="new">Are you sure you want to reveal the selected location(s)?</target>
+        <note>key: bulk_unhide.popup.message</note>
+      </trans-unit>
+      <trans-unit id="02b906429a29023075754b4dd3bf437b80a25ce6" resname="bulk_unhide.error.modal.table_title">
+        <source>Locations cannot be revealed (%itemsCount%)</source>
+        <target state="new">Locations cannot be revealed (%itemsCount%)</target>
+        <note>key: bulk_unhide.error.modal.table_title</note>
+      </trans-unit>
+      <trans-unit id="1208ce7816f3fa2b489c6946068c415949cf4c64" resname="bulk_unhide.error.message">
+        <source>%failedCount% of the %totalCount% selected location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Please contact your Administrator to obtain permissions.</source>
```

Comment:
```suggestion
        <source>%failedCount% of the %totalCount% selected Location(s) could not be revealed because you do not have proper user permissions. {{ moreInformationLink }} Contact your Administrator to obtain permissions.</source>
```